<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Hayabusa" ignored="0" name="FirstTest" tests="1" failures="1" timestamp="09 thg 6 2019 09:08:02 GMT" time="5.693" errors="0">
  <testcase name="TC001" time="0.938" classname="com.logigear.test.ta_dashboard.testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="Can&amp;apos;t login with correct credentials expected [SampleRepositoy] but found []">
      <![CDATA[java.lang.AssertionError: Can't login with correct credentials expected [SampleRepositoy] but found []
at com.logigear.test.ta_dashboard.testcases.LoginTest.TC001(LoginTest.java:26)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- TC001 -->
</testsuite> <!-- FirstTest -->
